src/data/real/ennreal.lean

```diff
@@ -1699,6 +1699,10 @@ lemma of_real_pow {p : ℝ} (hp : 0 ≤ p) (n : ℕ) :
   ennreal.of_real (p ^ n) = ennreal.of_real p ^ n :=
 by rw [of_real_eq_coe_nnreal hp, ← coe_pow, ← of_real_coe_nnreal, nnreal.coe_pow, nnreal.coe_mk]
 
+@[simp] lemma nsmul_of_real {x : ℝ} {n : ℕ} :
+  n • ennreal.of_real x = ennreal.of_real (n • x) :=
+by simp only [nsmul_eq_mul, ← of_real_coe_nat n, ← of_real_mul n.cast_nonneg]
```

Comment:
I would expect this to generalize to `n : nnrat` and `n : nnreal` in some reasonable way; I'll see what I can come  up with...

Reply:
OK sure, if I can find something that would be good. In the meantime, I've just kept the version for `ennreal`.